### ISSUE TITLE
Avoid underscores in URLs

### ISSUE THREAD

**Issue by gilleygoat** [web-infrastructure/doc_architecture#45]
Underscores should be avoided in URLs for a number of reasons. Use hyphens instead.

1.     Unlike hyphens, an underscore is not considered a word separator. This means the SEO value of keywords in a URL is undermined or lost.
2.     Underscores are hard for users to read. They merge into the underline of an underlined link, and are much harder for non-technically savvy users to type or share (not everyone knows what an underscore even is).

Hyphens have none of these drawbacks and should be used instead.
